src/ResourceManager/Network/Commands.Network/help/New-AzureRmVirtualNetworkGateway.md

```diff
@@ -210,6 +254,44 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -RadiusServerAddress
+P2S External Radius server address.```yaml
+Type: String
+Parameter Sets: RadiusServerConfiguration
+Aliases: 
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -RadiusServerSecret
+P2S External Radius server secret.```yaml
```

Comment:
You will need to break the line bfore (backtick)(backtick)(backtick)yaml

Reply:
Fixed - I am not sure why this was auto generated in this way.